backend/README.md

```diff
@@ -1,79 +1,65 @@
-To generate documenation used `npm run compodoc:build` to serve documentation use `npm run compodoc`
+`/backend` includes the server side code for running Mobiilinutakortti app.
```

Comment:
Nitpicking, but when referring to directories, starting with slash might be triggering. In this case it's a relative path, so ./backend is more suitable, or simply "The `backend` directory --".